rebar.config

```diff
@@ -97,6 +97,6 @@
         {overlay, [
             {mkdir, "log/sasl"},
             {mkdir, "data/"},
-            {copy, "bin/spartan-env", "bin"},
+            {template, "bin/spartan-env", "bin/spartan-env"},
```

Comment:
Does using this as a template mean that in the `ERTS_VSN="{{ erts_vsn }}"`, `{{ erts_vsn }}` will get filled with the `0.0.1` from
```
{relx, [{release, {spartan, "0.0.1"}, [spartan, erldns, observer, {eper, load}, {runtime_tools, load}, {eflame, load}, {debugger, load}]}
```
?